package.json

```diff
@@ -14,7 +14,7 @@
     "test:e2e": "wp-scripts test-e2e",
     "test:e2e:debug": "wp-scripts --inspect-brk test-e2e --puppeteer-devtools",
     "test:e2e:help": "wp-scripts test-e2e --help",
-    "test:e2e:watch": "npm run test-e2e -- --watch",
+    "test:e2e:watch": "npm run test:e2e -- --puppeteer-interactive",
```

Comment:
The `test:e2e` change makes sense, but do we want to change `--watch` to `--puppeteer-interactive`?

Reply:
I wasn't sure what the `--puppeteer-interactive` bit did exactly as `--watch` did make it do what I wanted it to. I thought that maybe they were synonymous.

Reply:
I think `--watch` changes the behavior of Jest so the tests run continuously every time you change a file. You'd get a screen like this:

<img width="640" alt="Screen Shot 2021-11-02 at 10 20 08 AM" src="https://user-images.githubusercontent.com/7188409/139819606-61926084-17c3-4d1e-9f05-4e39cd7f2422.png">

`--puppeteer-interactive` opens a Chromium window so you can see whatever is going on the test. In my opinion we should keep `test:e2e:watch` to `--watch` and if it is useful to have an alias for `--puppeteer-interactive`, we could add that.